examples/browser-video-streaming/streaming.js

```diff
@@ -20,3 +19,145 @@ document.addEventListener('DOMContentLoaded', async () => {
     hls.on(Hls.Events.MANIFEST_PARSED, () => video.play())
   }
 })
+
+// TODO: remove everything below here once https://github.com/ipfs/js-ipfs/pull/2683
+// and https://github.com/moshisushi/hlsjs-ipfs-loader/pull/16 are released
```

Comment:
Remove now?

Reply:
The `hlsjs-ipfs-loader` PR has been merged but not published yet so I don't think we can remove this yet.

Reply:
It got published! `0.2.3`